entanglement_forging/utils/forging_subroutines.py

```diff
@@ -129,7 +138,9 @@ def eval_forged_op_with_result(
     (Hamiltonian in the basis of determinants/bitstrings).
     For reference, also computes mean value obtained without Richardson
     """
-    tensor_state_prefixes = [f"bs{idx}" for idx in range(len(bitstrings_s))]
+    tensor_state_prefixes_u = [f"bsu{idx}" for idx in range(len(bitstrings_s_u))]
```

Comment:
This probably isn't necessary for this PR, but I will leave it in anticipation of U =/= V